.gitignore

```diff
@@ -40,3 +40,6 @@ cron.log
 # Ignore secret info
 /config/secrets.yml
 
+
```

Comment:
Remove figaro, put facebook secrets into credentials.yml.enc

Reply:
Gem figaro removed. I guess I need to delete the application.yml file and remove the path to it from .gitignore. (Because facebook secrets are kept in application.yml)